include/RAJA/index/ListSegment.hpp

```diff
@@ -223,6 +281,25 @@ class TypedListSegment
   //! accessor to retrieve the total number of elements in a TypedListSegment
   RAJA_HOST_DEVICE Index_type size() const { return m_size; }
 
+  //! Create a slice of this instance as a new instance but with no ownership
+  //! of the data it points to.
+  /*!
+   * \return A new instance spanning *begin() + begin to *begin() + begin +
+   * length
+   * 
+   * Current status: Tiling may not be distributing data correctly
```

Comment:
We should fix it... 😄

Reply:
Is it a "program might be slower than intended" or a "program will give wrong results silently" problem?

Reply:
The program will segfault if data is not owned when combining tiling with Kernel.